arduino-ide-extension/src/browser/boards/boards-service-provider.ts

```diff
@@ -364,12 +367,22 @@ export class BoardsServiceProvider implements FrontendApplicationContribution {
     return this._boardsConfig;
   }
 
+  get latestValidBoardsConfig():
+    | RecursiveRequired<BoardsConfig.Config>
+    | undefined {
+    return this._latestValidBoardsConfig;
+  }
+
+  get latestBoardsConfig(): BoardsConfig.Config | undefined {
+    return this._latestBoardsConfig;
+  }
```

Comment:
Do you use public APIs for the tests or do you think this will be used anywhere else in the app?

If the former, I propose using `protected` and subclass for the tests and increasing the visibility. If the latter, please explain where you plan to use it.